src/sparselm/model/_base.py

```diff
@@ -57,21 +88,15 @@ def __init__(
                 cvxpy backend solver to use. Supported solvers are:
                 ECOS, ECOS_BB, CVXOPT, SCS, GUROBI, Elemental.
                 GLPK and GLPK_MI (via CVXOPT GLPK interface)
```

Comment:
SCIP is also supported in a later update, with requirements that the user have compiled the SCIP executable in the system.

Reply:
I think I will remove this all-together and link to the cvxpy documentation so that we do not need to update this whenever a new solver is supported.